essentials-framework/Essentials Devices Common/Essentials Devices Common/VideoCodec/CiscoCodec/RoomPresets.cs

```diff
@@ -33,39 +28,19 @@ public interface IHasCodecRoomPresets
     public static class RoomPresets
     {
         /// <summary>
-        /// Converts Cisco RoomPresets to generic CameraPresets
+        /// Converts non-generic RoomPresets to generic CameraPresets
         /// </summary>
         /// <param name="presets"></param>
         /// <returns></returns>
-        public static List<CodecRoomPreset> GetGenericPresets(List<CiscoCodecStatus.RoomPreset> presets)
+        public static List<PresetBase> GetGenericPresets(List<IConvertiblePreset> presets)
         {
-            var cameraPresets = new List<CodecRoomPreset>();
-
-            if (Debug.Level > 0)
-            {
-                Debug.Console(1, "Presets List:");
-            }
-
-            foreach (CiscoCodecStatus.RoomPreset preset in presets)
-            {
-                try
-                {
-                    var cameraPreset = new CodecRoomPreset(UInt16.Parse(preset.id), preset.Description.Value, preset.Defined.BoolValue, true);
-
-                    cameraPresets.Add(cameraPreset);
+            Debug.Console(2, "Presets List:");
```

Comment:
Not much value in this statement...guessing it was meant to either be removed or have more information?

Reply:
Yeah - the debug?  It's meant to be gone, I just missed it.